search-app/src/cmr/search/services/query_execution.clj

```diff
@@ -170,7 +173,8 @@
   ;; We execute a base query with all the parameters to get the result and facets of fields that
   ;; are not in the query, then we merge this base result with only the facets for each individual
   ;; facet field that is in the query.
-  (let [facet-fields-in-query (keep #(when (facet-condition-resolver/has-field? query %) %)
+  (let [facet-fields-in-query (keep #(when (facet-condition-resolver/has-field? query %)
```

Comment:
I think `filter` would be clearer in this case and not go over the line length limit 😄 .
`(filter #(facet-condition-resolver/has-field? query %))`.